src/coreComponents/mesh/ElementRegionManager.cpp

```diff
@@ -693,6 +779,94 @@ ElementRegionManager::getCellBlockToSubRegionMap( CellBlockManagerABC const & ce
+
+      forElementRegions< CellElementRegion >( [&]( CellElementRegion const & elemRegion )
+      {
+        elemRegion.forElementSubRegions< CellElementSubRegion >( [&]( CellElementSubRegion const & subRegion )
+        {
+          printf( "  %s\n", subRegion.getName().c_str() );
+
+          CellElementSubRegion::NodeMapType const & elemToNodeRelation = subRegion.nodeList();
+          arrayView2d< localIndex const, cells::NODE_MAP_USD > const elemToNode = elemToNodeRelation;
+          arrayView1d< globalIndex const > const & elemLocalToGlobal = subRegion.localToGlobalMap();
+          auto const & elemGlobalToLocal = subRegion.globalToLocalMap();
+          arrayView1d< globalIndex const > const & nodeLocalToGlobal = elemToNodeRelation.relatedObjectLocalToGlobal();
+//          auto const & refCoords = getParent().getGroup< NodeManager >( "nodeManager" ).referencePosition();
+
+          printf( "  ElementToNodes map:\n" );
+          for( localIndex k=0; k<elemToNode.size( 0 ); ++k )
+          {
+            printf( "  %3d( %3lld ): ", k, elemLocalToGlobal( k ) );
+            for( localIndex a=0; a<elemToNode.size( 1 ); ++a )
+            {
+              printf( "%3d", elemToNode( k, a ) );
+              if( a != elemToNode.size( 1 )-1 )
+              {
+                printf( ", " );
+              }
+            }
+            printf( " )\n" );
+          }
+
+          printf( "\n  ElementToNodes map ( global nodes sorted by global elems):\n" );
+          map< globalIndex, localIndex > const sortedGlobalToLocalMap( elemGlobalToLocal.begin(),
+                                                                       elemGlobalToLocal.end());
+          for( auto indexPair : sortedGlobalToLocalMap )
+          {
+            globalIndex const gk = indexPair.first;
+            localIndex const k = indexPair.second;
+
+            printf( "  %3d( %3lld ): ", k, gk );
+            for( localIndex a=0; a<elemToNode.size( 1 ); ++a )
+            {
+              printf( "%3lld", nodeLocalToGlobal( elemToNode( k, a ) ) );
+              if( a != elemToNode.size( 1 )-1 )
+              {
+                printf( ", " );
+              }
+            }
+            printf( " )\n" );
+          }
+
+          // printf( "\n  ElementToNodes coords:\n" );
+          // for( auto indexPair : sortedGlobalToLocalMap )
+          // {
+          //   globalIndex const gk = indexPair.first;
+          //   localIndex const k = indexPair.second;
+
+          //   printf( "  %3d( %3lld ): ", k, gk );
+          //   for( localIndex a=0; a<elemToNode.size( 1 ); ++a )
+          //   {
+          //     localIndex const b = elemToNode( k, a );
+          //     printf( "( %4.1f, %4.1f, %4.1f )", refCoords( b, 0), refCoords( b, 1 ), refCoords( b, 2 ) );
+          //     if( a != elemToNode.size( 1 )-1 )
+          //     {
+          //       printf( ", " );
+          //     }
+          //   }
+          //   printf( " )\n" );
+          // }
```

Comment:
can this be deleted?

Reply:
this stuff is handy to have around in case we need to dump out all this info at some point. I think I may need it for the follow on PR.